crates/aptos-telemetry/src/build_information.rs

```diff
@@ -29,32 +29,37 @@ pub const BUILD_VERSION: &str = "build_version";
 #[macro_export]
 macro_rules! collect_build_information {
     () => {{
+        println!("calling collect {:?}", std::env::var("GIT_SHA"));
```

Comment:
I'm assuming we'll want to remove this :)

Reply:
@ibalajiarun Please make a PR to remove this so it doesn't print on every startup / usage.  I didn't realize auto merge was on